pom.xml

```diff
@@ -1276,6 +1276,7 @@
     <collections.version>3.2.2</collections.version>
     <httpclient.version>3.1</httpclient.version>
     <httpcore.version>4.4.4</httpcore.version>
+    <http-components-client.version>4.5.2</http-components-client.version>
```

Comment:
Is this the version we currently package in 1.4.x?

Reply:
There were three different versions of org.apache.httpcomponents:httpclient in branch-1.4 namely: 4.1.2, 4.2.5 and 4.5.2. Should I use one of the other two versions?